openbb_terminal/stocks/tradinghours/tradinghours_controller.py

```diff
@@ -231,3 +239,45 @@ def call_all(self, other_args: List[str]):
         ns_parser = self.parse_known_args_and_warn(parser, other_args)
         if ns_parser:
             bursa_view.display_all()
+
+    @log_start_end(log=logger)
+    def call_holidays(self, other_args: List[str]):
+        """Process 'holidays' command."""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="exchange",
+            description="Select the exchange and year you want see holidays for",
+        )
+        parser.add_argument(
+            "-n",
+            "--name",
+            help="Exchange short name",
+            dest="exchange",
+        )
+
```

Comment:
I see what happened.  We did a refactor of the way that the autocomplete is defined so that choices are taken from the controller arguments (so that they only need to be defined once which helps us maintain).  So in order to get the correct autocomplete, you need to add choices = self.all_holiday_exchange_short_names to the n argument.